### DISCUSSION TITLE
EN Batch is inconsistent with L1

### DISCUSSION THREAD

**Discussion by sshamanov** [zkSync-Community-Hub/zksync-developers#249] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

Have an error in External Node: Batch is inconsistent with L1

### What is the specific issue or error you're encountering?

ERROR zksync_utils::wait_for_tasks: One of the tokio actors unexpectedly finished with error: Batch 363174 is inconsistent with L1

### Can you share the error messages or logs you're receiving, if any?

ERROR zksync_utils::wait_for_tasks: One of the tokio actors unexpectedly finished with error: Batch 363174 is inconsistent with L1

### Have you made any recent changes to the contract before encountering this issue?

No

### Are there any external libraries or contracts that your contract interacts with?

No

### Can you provide the relevant portions of your contract code where the issue is occurring?

No

### Have you tried to isolate the problem, and if so, what were the results?

No

### What steps have you already taken to try to resolve the issue?

Tried to upgrade, but it is no updated container version
Tried to change RPC endpoint

**Comment by githubdoramon** [comment, marked as answer]
What version are you using of the node?

There is a new version released today (18.10) due to a bug discovered around 5:50 UTC. Have you tried this one already?

**Reply by sshamanov** [reply]
It was 18.8.0
Upgrade to 18.10.1 solved the problem
Thank you

**Comment by inpaslimabj** [comment]
The error message "Batch is inconsistent with L1" typically indicates a synchronization issue between zkSync's Layer 1 and Layer 2. This could occur due to various reasons such as network disruptions, data corruption, or inconsistencies in the batch processing.

To resolve this issue:
1. Ensure that both Layer 1 and Layer 2 nodes are running smoothly without any errors or disruptions.
2. Check for any network connectivity issues or congestion that might be affecting data synchronization.
3. Verify that the RPC endpoint is correctly configured and reachable.
4. If possible, try restarting the affected components or nodes to see if the issue resolves itself.
5. If the problem persists, consider reaching out to the zkSync development team for further assistance and troubleshooting.

Additionally, monitoring the logs and system status closely can provide insights into the root cause of the inconsistency and help in resolving it effectively.

**Comment by batesb188** [comment]
Encountered an error in the External Node: "Batch is inconsistent with L1". Attempted to resolve by upgrading and changing RPC endpoint, but issue persists. Further investigation needed.

**Comment by tashuraschaefer** [comment]
The error "Batch is inconsistent with L1" typically indicates a synchronization issue between zkSync's layer 1 and layer 2. This could be due to various reasons such as network congestion, data corruption, or misconfiguration. I would recommend reaching out to the zkSync support team for assistance in resolving this issue.

**Comment by msandiferv** [comment]
The error "Batch is inconsistent with L1" indicates a synchronization issue between zkSync and the Ethereum L1 network. 
This could be due to various reasons such as network congestion, chain reorganizations, or node misconfigurations. 
To resolve it, consider checking network status, ensuring proper node configuration, and consulting zkSync documentation or support for further assistance.

**Comment by m4ccmocare** [comment]
The user is encountering an error in their external node, specifically "Batch is inconsistent with L1". They have not made any recent changes to their contract and are not interacting with any external libraries or contracts. They have tried to resolve the issue by upgrading and changing the RPC endpoint, but have not been successful. No repository link was provided.
